app/controllers/students_controller.rb

```diff
@@ -0,0 +1,59 @@
+class StudentsController < ApplicationController
+  # TODO remove below when data base set
+  before_action :set_images
+
+  def index
+    @page_title = 'Students Profiles'
```

Comment:
for the page title, should students be singular (student profiles vs students profiles) ?